app/assets/javascripts/client/home/StartAssessmentCtrl.js

```diff
@@ -0,0 +1,97 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('StartAssessmentCtrl', StartAssessmentCtrl);
+
+  StartAssessmentCtrl.$inject = [
+    '$scope',
+    '$timeout',
+    '$location',
+    '$modal',
+    'Assessment',
+    'SessionService'
+  ];
+
+  function StartAssessmentCtrl($scope, $timeout, $location, $modal, Assessment, SessionService) {
+    var vm = this;
+
+    vm.user = SessionService.getCurrentUser();
+    vm.district = vm.user.districts[0];
+
+    vm.alerts = [];
+    vm.assessment = {};
+
+    vm.hideModal = function() {
+      $('#startAssessment').modal('hide');
+    };
+
+    vm.noDistrict = function() {
+      return _.isEmpty(vm.user.district_ids);
+    };
+
+    vm.redirectToAssessment = function(assessment) {
+      vm.hideModal();
+      $location.path('assessments/' + assessment.id + '/assign');
+    };
+
+    vm.create = function(assessment) {
+      assessment.due_date = moment($("#due-date").val(), 'MM-DD-YYYY').toISOString();
```

Comment:
I understand that, but honoring the same spirit of previous code-reviews in this same project, it's likely we'd like to fix this now since you previously suggested fixes on code that was being borrowed from other parts of the project that don't always feel entirely right. I'm not suggesting fixing other parts of the projects with this new service, I'm just suggesting creating a service and use it here now, this way another developer who wants to borrow your code doesn't have to fix it/duplicate it.